packages/react-components/react-table/src/hooks/useTableColumnSizing.tsx

```diff
@@ -77,14 +77,16 @@ function useTableColumnSizingState<TItem>(
       getColumnWidths: columnResizeState.getColumns,
       getTableHeaderCellProps: (columnId: TableColumnId) => {
         const col = columnResizeState.getColumnById(columnId);
+        const isLastColumn = columns[columns.length - 1]?.columnId === columnId;
 
-        const aside = (
+        const aside = isLastColumn ? null : (
```

Comment:
having this here for the last columns means that it _always_ overflows it's parent. Since it is not needed (it doesn't do anything), just get rid of it altogether.